scrapers/USA/KY/index.js

```diff
@@ -0,0 +1,54 @@
+import csvParse from 'csv-parse';
+import * as fetch from '../../../lib/fetch.js';
+import * as parse from '../../../lib/parse.js';
+import * as transform from '../../../lib/transform.js';
+import * as geography from '../../../lib/geography.js';
+
+const scraper = {
+  country: 'USA',
+  state: 'KY',
+  priority: 1,
+  url: 'https://datawrapper.dwcdn.net/BbowM/23/',
+  source: {
+    name: 'Kentucky Cabinet for Health and Family Services',
+    url: 'https://www.kentucky.com/news/coronavirus/article241309406.html'
+  },
+  maintainers: [
+    {
+      name: 'Jordan Holt',
+      email: 'jordholt@gmail.com',
+      github: 'DatJord',
+      country: 'USA'
+    }
+  ],
+  aggregate: 'county',
+  async scraper() {
+    const $ = await fetch.headless(this.url);
+
+    const counties = [];
+
+    // Extract raw csv from link attribute on Kentucky Health Organizations data map.
+    const csvText = decodeURIComponent($('a[class="dw-data-link"]').attr('href')).replace('data:application/octet-stream;charset=utf-8,', '');
+
+    // Parse csv.
+    const data = csvParse(csvText, { columns: true });
```

Comment:
Go ahead and make this async: https://csv.js.org/parse/api/callback/

Reply:
Or if that's annoying, you can do it sync too :) https://csv.js.org/parse/api/sync/